### PR TITLE
Delivery of M3 of UOI project

### PR DESCRIPTION
**Pull request by cjdelisle**
Hey folks, we have a milestone for UOI project. Actually this was completed a while back but it got stuck in my backlog.

### CODE REVIEW

**Review by cjdelisle — COMMENTED** [review]
I will abstain from approving this PR since it is my project.

**Review by Arceliar — APPROVED** [review]
In theory, for the UOI fund, I'd like to see projects that are a bit more closely related to PKT. That said, I do like all of these projects, and on average I think the new projects are more PKT-relevant than the ones from the previous milestone (so things are arguably trending in the right direction). Looks good to me!

File: projects/2021_05_09_user_operated_internet_fund.md [hunk 1]
```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
...
+
+- 20mn PKT was converted into € 377 240. See #4.
+
+The current balance is 0 PKT.
+
+### 2. Converted from PKT
+
+The amount of money which was converted from PKT during the reported period
+
+    10mn PKT -> 159740,16 (180.000 dollar)
```
Comment (dvn0) [inline projects/2021_05_09_user_operated_internet_fund.md:541]: Can this be changed to match the formatting used throughout the  rest of the document? eg. `$180,000`

File: projects/2021_05_09_user_operated_internet_fund.md [hunk 2]
```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
...
+* Assets as of March 8th 2022
+
+  * Total available assets: 0 PKT, 377,240€
+  * 325,802 € budget allocated to regular projects (93.8%)
+    * Spent: 28 200 €
+  *  10,000 € (of max 113,172 €) allocated to support projects (2.7%, max 30%)
+    * Spent: 0€
+  *  13,238 € administrative budget (fixed 10%, temporarily reduced to 3.7%)
+    * Spent: 20,163€
+    * Prefunded by NLnet; 6925€
```
Comment (dvn0) [inline projects/2021_05_09_user_operated_internet_fund.md:564]: The currency format is non consistent in this section. Should all be €<amount>  to match the rest of the doc.

File: projects/2021_05_09_user_operated_internet_fund.md [hunk 3]
```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
...
+https://nlnet.nl/thema/User-operatedInternetFund.html
+
+### 7. Support projects
+
+For each active funded project which is classified as a support project: a management overview of the use of resources, remaining budget, and which projects were supported.
+
+* **Radically Open Security - Security scans, audits and advice**
+  - Use of resources:
+      * 0€ claimed
+      * Remaining budget: 10 000€
```
Comment (dvn0) [inline projects/2021_05_09_user_operated_internet_fund.md:679]: Place the € at the front of amount

File: projects/2021_05_09_user_operated_internet_fund.md [hunk 4]
```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
...
+### 8. Attestation
+
+I attest on my honor that the above transaction information is true and accurate and that the payments to projects have been made after the agreed results have been delivered
+  * **signature on PDF version of this document**
+ 
+### 9. Metrics
+
+Metrics regarding applications to the 2022-02U call:
+* Number of applications: **12**
+* Total budget requested: **480,389**€
```
Comment (dvn0) [inline projects/2021_05_09_user_operated_internet_fund.md:690]: Place the € at the front of amount.

**Review by dvn0 — COMMENTED** [review]
Barring the minor cosmetic changes I suggested in the comments, this all looks good to me.

File: projects/2021_05_09_user_operated_internet_fund.md (hunk 4 above)
Comment (cjdelisle) [reply projects/2021_05_09_user_operated_internet_fund.md:690]: In Europe, the € is placed after the number. But anyway these types of issues are not what we're looking for, more like anything which could be used to argue that the payment is unjustified.

**Pull request merged**